packages/messages/fixtures/chatmessages.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
@JoseNoriegaa why don't we use Faker to create these fixtures and in this way we avoid to hardcode the data which is an antipattern in testing?

Reply:
As we discuss on slack, this is going to be made in #139